packages/webapp/src/lib/services/loadRegistryUrl.ts

```
@@ -0,0 +1,95 @@
+	}));
+
+	vi.mock('@rainlanguage/ui-components/services', () => ({
+		fetchRegistryDotrains: vi.fn()
+	}));
+
+	describe('loadRegistryUrl', () => {
+		beforeEach(() => {
+			vi.resetAllMocks();
+			// Reset window.location
+			// eslint-disable-next-line @typescript-eslint/no-explicit-any
+			delete (global.window as any).location;
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)